book/section4/mediator.md

```diff
@@ -2,22 +2,132 @@
 
 ## Overview
 
-A Self-Sovereign Identity Mediator is a service that enables private and secure connections between peers. Its primary function is to relay encrypted messages between DIDs using [DIDComm V2](../section3/didcomm.md) protocols. Since each participant wallet is Self-Sovereign, Mediators help route messages to their intended recipient without any one wallet knowing the details of the other.  Encrypted messages can be sent to peers even if they are not online, and the Mediator will manage a queue for the offline party until they are able to connect and retrieve them.
+The `mediator` is a service that enables private and secure message routing between peers. Its primary function is to relay encrypted messages between agents using [DIDComm V2](../section3/didcomm.md) set of communication protocols. Since each participant wallet is Self-Sovereign, mediators help route messages to their intended recipient without any details of each peer besides the agreed public keys over the initial connection protocol, upon which they both create a `did:peer`.  Encrypted messages can be sent to peers even if they are not online by relaying on the mediator to manage a queue for the offline party until they are able to connect and retrieve them.
```

Comment:
- `using DIDCommV2 communication protocols` (no need for `set of`)

Reply:
Capitalize `The Mediator`

Reply:
suggestion:

Since each participant wallet is Self-Sovereign, mediators help route messages to their intended recipient without any details of each peer besides the connected public keys of each participant.  The pairing of each DID in a Connection is known as a `did:peer`.

(I think my suggestion here may get this wrong).  I think my comment is that the original sentence feels like it could be broken up in to two parts.  So if there's a way to do that, ignore my suggested replacement.